### PR TITLE
docs: add --with-sys-table detail description

### PR DESCRIPTION
**Pull request by benmaoer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/br-snapshot-manual.md [hunk 1]
```diff
@@ -145,22 +146,25 @@ br backup full\
...
     --storage "s3://${backup_collection_addr}/snapshot-${date}?access-key=${access-key}&secret-access-key=${secret-access-key}" \
     --ratelimit 128 \
     --log-file restorefull.log
 ```
 
 以上命令中，
 
-- `--ratelimit`：**每个 TiKV** 执行恢复任务的速度上限（单位 MiB/s）
-- `--log-file`：备份日志写入的目标文件
+- `--with-sys-table`: 恢复集群数据的同时恢复 **部分系统表** 的数据，包括恢复账号权限数据和 SQL Binding 信息，但暂不支持恢复统计信息 (`mysql.stat_*`) 和系统参数 (`mysql.tidb, mysql.global_variables`) 等信息，更多详细信息见[恢复 mysql 数据库下的表](/br/br-snapshot-guide.md#恢复-mysql-数据库下的表)。
```
Comment (lilin90) [inline br/br-snapshot-manual.md:157]:
```suggestion
- `--with-sys-table`：恢复集群数据的同时恢复**部分系统表**的数据，包括恢复账号权限数据和 SQL Binding 信息，但暂不支持恢复统计信息 (`mysql.stat_*`) 和系统参数 (`mysql.tidb`, `mysql.global_variables`) 等信息，更多信息详见[恢复 mysql 数据库下的表](/br/br-snapshot-guide.md#恢复-mysql-数据库下的表)。
```

File: br/br-snapshot-manual.md [hunk 2]
```diff
@@ -145,22 +146,25 @@ br backup full\
...
+- `--ratelimit`：**每个 TiKV** 执行恢复任务的速度上限（单位 MiB/s）。
+- `--log-file`：备份日志写入的目标文件。
 
 恢复期间终端会显示进度条，效果如下。当进度条达到 100% 时，表示恢复完成。在完成恢复后，br 工具为了确保数据安全性，还会校验恢复数据。
 
 ```shell
 Full Restore <---------/...............................................> 17.12%.
 ```
 
+
```
Comment (lilin90) [inline br/br-snapshot-manual.md:167]: 请删除多余空行，文内标题上方保留一个空行即可。

File: br/br-snapshot-manual.md [hunk 3]
```diff
@@ -145,15 +145,17 @@ br backup full\
...
     --storage "s3://${backup_collection_addr}/snapshot-${date}?access-key=${access-key}&secret-access-key=${secret-access-key}" \
     --ratelimit 128 \
     --log-file restorefull.log
 ```
 
 以上命令中，
 
-- `--ratelimit`：**每个 TiKV** 执行恢复任务的速度上限（单位 MiB/s）
-- `--log-file`：备份日志写入的目标文件
+- `--with-sys-table`：恢复集群数据的同时恢复**部分系统表**的数据，包括恢复账号权限数据和 SQL Binding 信息，但暂不支持恢复统计信息 (`mysql.stat_*`) 和系统参数 (`mysql.tidb`, `mysql.global_variables`) 等信息，更多信息详见[恢复 mysql 数据库下的表](/br/br-snapshot-guide.md#恢复-mysql-数据库下的表)。
```
Comment (lilin90) [inline br/br-snapshot-manual.md:156]:
```suggestion
- `--with-sys-table`：恢复集群数据的同时恢复**部分系统表**的数据，包括恢复账号权限数据和 SQL Binding 信息，但暂不支持恢复统计信息 (`mysql.stat_*`) 和系统参数 (`mysql.tidb`, `mysql.global_variables`) 等信息，更多信息详见[恢复 `mysql` 数据库下的表](/br/br-snapshot-guide.md#恢复-mysql-数据库下的表)。
```

**Review by 3pointer — APPROVED** [review]

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**